docs/source/azure/usecases/lbfilterbyfrotendip.rst

```diff
@@ -0,0 +1,14 @@
+Filter load balancer by front end public ip
+============================
```

Comment:
this will probably fail on "insufficient underline" or whatever the error is about the underline not being as long as the text above it